### DISCUSSION TITLE
Allow Individual Meetings to Assign a District

### DISCUSSION THREAD

**Discussion by dcfnet** [code4recovery/12-step-meeting-list#669] [Q&A] [answered]
Hello!

As of now, the only way I can assign a meeting to a District is to select 'Group' as opposed to 'Individual Meeting'. All 350+ of the meetings in our area are divided into four Districts. When I assign the meeting to a Group, then District, I lose the ability to provide Contact Info for that meeting.  Or the Contact Info for that meeting is populated with the info for the District.

Although I can't do this from the web interface when editing or adding a meeting, I can add Contact, PayPal, Email, etc to the downloaded CSV and upload it to the website.

It would be handy to be able to assign a District and also unique contact info to a particular meeting. Make sense?

Thank you all for your continued excellent service!

David
(757) 652-1711

**Comment by joshreisner** [comment]
Hi David! Seems like a question for the WordPress 12 Step Meeting List plugin, going to move the issue there.

**Comment by dcfnet** [comment]
*** UPDATE ***
The editing and uploading of the .CSV actually doesn't work like I thought. Same basic result. If I add email, phone, PayPal, Venmo, etc. to one meeting, that information gets associated with every meeting in that District.  We really want all meetings to sort by District. But, we also really want to assign data unique to that meeting... At the moment, we are opting in for the District sort and forgoing the ability to add contact info.

**Comment by dcfnet** [comment]
The DISTRICT designation is important. But, within each District, is really a collection of Individual Meeting.
(just trying to clarify - I don't write good)

**Comment by joshreisner** [comment, marked as answer]
try giving a unique value in each group name. maybe add the day and time to the end of it

**Comment by dcfnet** [comment]
...I'll try it. Thank you!

**Comment by dcfnet** [comment]
THAT looks like `it gives us exactly what we need! Thanks Josh!

**Reply by joshreisner** [reply]
great!
